src/ai/backend/manager/api/session.py

```diff
@@ -383,10 +382,10 @@ async def _create(request: web.Request, params: dict[str, Any]) -> web.Response:
             tx.AliasedKey(["name", "session_name", "clientSessionToken"], default=undefined)
             >> "session_name": UndefChecker | t.Regexp(r"^(?=.{4,64}$)\w[\w.-]*\w$", re.ASCII),
             tx.AliasedKey(["image", "lang"], default=undefined): UndefChecker | t.Null | t.String,
-            tx.AliasedKey(["arch", "architecture"], default=DEFAULT_IMAGE_ARCH)
-            >> "architecture": t.String,
-            tx.AliasedKey(["type", "sessionType"], default="interactive")
-            >> "session_type": tx.Enum(SessionTypes),
```

Comment:
when we create session template, these values already set default value here 
https://github.com/lablup/backend.ai/blob/4c62f7c38431d8f9fcdd52959b26f1d60b1ee05e/src/ai/backend/manager/models/session_template.py#L60-L65